### DISCUSSION TITLE
Can I read/write ceph from external cluster?

### DISCUSSION THREAD

**Discussion by mc956759** [rook/rook#11536] [Q&A] [answered]
I deploy ceph cluster by k8s+rook, and I need read/write from external by librados. 
Librados just can visit the port 6789 of mon, but I don't know how to expose port 6789 of mon pod, or how to config librados to specify port, which exposing by k8s service of mon pod.
Anyone can help me?

**Comment by Madhu-1** [comment]
@mc956759, when ceph cluster is deployed mon, uses clusterIP service. you need to external access to the ceph cluster you can create a ceph cluster on HostNetworking as mon uses NodePort.

**Reply by mc956759** [reply]
Thanks for your answer! 
And, how can I deploy mon on HostNetworking by rook? Any configure can support this operation?

**Comment by Madhu-1** [comment, marked as answer]
https://rook.io/docs/rook/v1.10/Storage-Configuration/Advanced/ceph-configuration/?h=host#use-hostnetwork-in-the-cluster-configuration will help you

**Reply by mc956759** [reply]
It works! Thank you very much.
